### PR TITLE
fix: adapt uds server auxiliary

### PR DESCRIPTION
**Pull request by yannpoupon**

### CODE REVIEW

File: src/pykiso/lib/auxiliaries/udsaux/uds_server_auxiliary.py [hunk 1]
```diff
@@ -126,14 +126,14 @@ def transmit(
         data = self._pad_message(data)
         self.channel._cc_send(msg=data, remote_id=req_id)
 
-    def receive(self) -> Optional[bytes]:
+    def receive(self, timeout: float = 0) -> Optional[bytes]:
         """Receive a message through ITF connector. Called inside a thread,
         this method is a substitute to the reception method used in the
         python-uds package.
```
Comment (sebastianpfischer) [inline src/pykiso/lib/auxiliaries/udsaux/uds_server_auxiliary.py:133]: parameter description missing

**Review by Pog3k — APPROVED** [review]

**Review by sebastianpfischer — APPROVED** [review]

**Comment by sebastianpfischer** [comment]
Somehow our ci/cd pipeline keeps failing. Please double-check the state of the coverage before clicking the merge button.

**Comment by yannpoupon** [comment]
> Somehow our ci/cd pipeline keeps failing. Please double-check the state of the coverage before clicking the merge button.

The pipeline is failing on master branch also with a test failing: https://ci.eclipse.org/kiso-testing/job/kiso-testing/job/master/250/

**Comment by sebastianpfischer** [comment]
> > Somehow our ci/cd pipeline keeps failing. Please double-check the state of the coverage before clicking the merge button.
> 
> The pipeline is failing on master branch also with a test failing: https://ci.eclipse.org/kiso-testing/job/kiso-testing/job/master/250/

@yannpoupon , there is a parallel PR to review to fix that.

**Pull request merged**